docs: Explain integration of external test results (#5735)
os-autoinst/openQA: docs/WritingTests.asciidoc

   it could be delivered. If you have gotten a 200 response then everything
   is setup correctly. Otherwise, checkout the response of the delivery to
   investigate what is wrong.

== Integrating test results from external systems

> The openQA webUI is very suitable as a test management and reporting platform.

Reviewer: ```suggestion
The openQA web UI is very suitable as a test management and reporting platform.
```

Author: done

> The openQA webUI is very suitable as a test management and reporting platform.
> Next to the automated openQA tests one can integrate test results from
> external systems or even manual test results by simply selecting a
> corresponding worker class for which no worker needs to exist at all. Example
> call to `openqa-cli` to create a test job with name "my_manual_test" on a
> local openQA instance:

Reviewer 2: ```suggestion
The openQA webUI is suitable as a test management and reporting platform.
Next to the automated openQA tests one can integrate test results from
external systems or manual test results by selecting a
worker class without a worker assigned to it. The following call 
to `openqa-cli` creates a test job with the name "my_manual_test" on a
local openQA instance:
```
